health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/AssumptionsList.js

```
@@ -23,13 +25,35 @@ const AssumptionsList = ({ customProps, setupCompleted }) => {
     return (
-        <div>
+        <div style={{ marginBottom: "2.5rem" }}>
+            <Card className="middle-child">
+                <Header className="summary-main-heading">{t(`MICROPLAN_ESTIMATION_ASSUMPTIONS_HEADING`)} </Header>
+                {orderedKeys.map((key, index) => {
+                    const assumption = campaignAssumption?.[key] || {};
+                    const code = assumption.code || "NA";
+                    const value = assumption.value || "NA";
+                    return (
+                        <>
+                            <LabelFieldPair className="as-label-field" style={{ marginBottom: "0px" }}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)